src/frontend/api/http/axios.ts

```diff
@@ -25,10 +27,19 @@ export const håndterApiRespons = <T>(apiRespons: ApiRespons<T>): Ressurs<T> =>
         innloggetSaksbehandler,
         loggFeilTilSentry = false,
         ressurs,
+        httpStatus,
     } = apiRespons;
 
     let typetRessurs: Ressurs<T>;
 
+    if (erServerFeil(httpStatus)) {
+        return {
+            frontendFeilmelding:
+                'En teknisk feil på våre servere gjør at siden er utilgjengelig. Dette skyldes ikke noe du gjorde. ',
```

Comment:
Megapirk + er ordet "servere" for vanskelig for brukere? Går det an å si "En feil hos oss gjør at siden er utilgjengelig. Det skyldes ikke noe du gjorde."
```suggestion
                'En teknisk feil på våre servere gjør at siden er utilgjengelig. Dette skyldes ikke noe du gjorde.',
```

Reply:
Vanskelig å si. Teksten er hentet fra Aksel, men vi kan godt endre den slik at det er enklest mulig for brukeren uansett.